server/apps/immich/src/config/asset-upload.config.ts

```diff
@@ -48,13 +46,28 @@ export function customStorage(): StorageEngine {
 
 export const multerUtils = { fileFilter, filename, destination };
 
+const logger = new Logger('AssetUploadConfig');
+
+function getMimeType(file: Express.Multer.File) {
+  const extension = file.originalname.split('.').pop() as string;
+  switch (extension.toLowerCase()) {
+    case 'raf':
+      return 'image/x-fuji-raf';
+    case 'srw':
+      return 'image/x-samsung-srw';
+    default:
+      return file.mimetype;
+  }
+}
+
```

Comment:
I don't think we need this anymore since the web is sending the correct mimetype now, right?

Otherwise looks good.

Reply:
Ah, you're right. Got rid of that. :)

Reply:
Actually, it is still required.